src/plugins/reset/reset-button.less

```
@@ -0,0 +1,14 @@
+.uip-reset {
+  display: inline-flex;
+  cursor: pointer;
+
+  > svg {
+    fill: currentColor;
+    width: 100%;
+    height: 100%;
+  }
+
+  &-hidden {
```

display: none - is just our version of customization, we have this button disabled actually, so there it will not be wrong to use disabled attribute even with the current styles.
```suggestion
  &[disabled] {
```